recordings/admin.py

```diff
@@ -24,12 +24,12 @@ class RecordingAdmin(admin.ModelAdmin):
         'public',
         ('dialect', admin.RelatedOnlyFieldListFilter),
         ('place', admin.RelatedOnlyFieldListFilter),
-        'place__country',
+        # 'place__country',
```

Comment:
Remove?

Reply:
Good catch. I wanted to re-enable this later, but I forgot.